lib/src/main/java/io/ably/lib/rest/Auth.java

```diff
@@ -176,6 +186,24 @@ public static AuthOptions copyAttributes(AuthOptions src, AuthOptions dest) {
 			src.authCallback = dest.authCallback;
 			return src;
 		}
+
+		/**
+		 * A special case convenience for AuthOption
+		 * that if all its attributes are null apart from force,
+		 * then when passed to authorise as an argument
+		 * the previously configured authentication options will remain intact
+		 * This behaviour takes precedence over RSA10j
+		 * <p>
+		 * Spec: RSA10d
+		 * </p>
```

Comment:
@paddy Why are we doing this? I can't see anything in the spec that says to do this, especially not RSA10d.